addon/components/pie-chart.js

```diff
@@ -482,10 +482,23 @@ export default ChartComponent.extend(FloatingTooltipMixin,
     var groups = this.get('groups').attr(this.get('groupAttrs'));
     groups.select('path').attr(this.get('sliceAttrs'));
     
-    var labelWidth = this.get('labelWidth');
+    // Maximum allowed width for a label is `outer width of the chart/2` - `required label padding`
```

Comment:
This comment isn't useful. It's more or less a word for word restatement of what the code says.